packages/openapi-validator/lib/utils/OpenApi3Spec.utils.js

```diff
@@ -0,0 +1,91 @@
+const generateCombinations = require('combos');
```

Comment:
This file could definitely be simpler, but I'm not sure how. Will be very grateful for suggestions! The tricky thing is to provide error messages with server urls that are templated before the path, and concrete after the path, e.g. `https://{hostVariable}.com:{portVariable}/defaultValueOfVariableInPath` (See the tests)